libs/spark/src/Unstable_Button/Unstable_Button.tsx

```diff
@@ -0,0 +1,277 @@
+          backgroundColor: darken(theme.unstable_palette.red[700], 0.2),
+        },
+        '&[aria-expanded="true"]': {
+          backgroundColor: theme.unstable_palette.neutral[600],
+        },
+        '&$disabled': {
+          backgroundColor: theme.unstable_palette.neutral[80],
+          color: theme.unstable_palette.neutral[100],
+        },
+      }),
+      ...(props.size === 'small' && {
+        ...buttonFontVariantSmall,
+        padding: '8px 16px',
+      }),
+      ...(props.size === 'medium' && {
+        ...buttonFontVariantMedium,
+        padding: '12px 24px',
+      }),
+      ...(props.size === 'large' && {
+        ...buttonFontVariantLarge,
+        padding: '20px 32px',
+      }),
+    }),
+
+    label: (props: Unstable_ButtonProps) => ({
+      color: 'inherit',
+      fontSize: 'inherit',
+      lineHeight: 'inherit',
+      ...(props.variant === 'primary' && {
+        color: theme.palette.common.white,
+        '&$disabled': {
+          color: theme.unstable_palette.neutral[100],
+        },
+      }),
+      ...(props.variant === 'stroked' && {
+        color: theme.unstable_palette.brand.blue,
+        '&[aria-expanded="true"]': {
+          color: theme.palette.common.white,
+        },
+      }),
+      ...(props.variant === 'ghost' && {
+        color: theme.unstable_palette.brand.blue,
+        '&[aria-expanded="true"]': {
+          color: theme.palette.common.white,
+        },
+        '&$disabled': {
+          color: theme.unstable_palette.neutral[100],
+        },
+      }),
+      ...(props.variant === 'destructive' && {
+        color: theme.palette.common.white,
+        '&$disabled': {
+          color: theme.unstable_palette.neutral[100],
+        },
+      }),
+    }),
+    startIcon: (props: Unstable_ButtonProps) => ({
+      display: 'inherit',
+      // -2px accounts for label vs icon size difference
+      // have to specify individually to override defaults
+      margin: '-2px',
+      marginBottom: '-2px',
+      marginRight: '8px',
+      marginLeft: 0,
+      color: 'inherit',
+      ...(props.size === 'small' && {
+        fontSize: theme.typography.pxToRem(16),
+        lineHeight: theme.typography.pxToRem(16),
+      }),
+      ...(props.size === 'medium' && {
+        fontSize: theme.typography.pxToRem(24),
+        lineHeight: theme.typography.pxToRem(24),
+      }),
+      ...(props.size === 'large' && {
+        fontSize: theme.typography.pxToRem(24),
+        lineHeight: theme.typography.pxToRem(24),
+      }),
+    }),
+    endIcon: (props: Unstable_ButtonProps) => ({
+      display: 'inherit',
+      // -2px accounts for label vs icon size difference
+      // have to specify individually to override defaults
+      marginTop: '-2px',
+      marginBottom: '-2px',
+      marginLeft: '8px',
+      marginRight: 0,
+      color: 'inherit',
+      ...(props.size === 'small' && {
+        fontSize: theme.typography.pxToRem(16),
+        lineHeight: theme.typography.pxToRem(16),
```

Comment:
```suggestion
      color: 'inherit',
      lineHeight: 1,
      ...(props.size === 'small' && {
        fontSize: theme.typography.pxToRem(16),
```
Line height's should be unitless. They become a coefficient of font size.